flask_potion/contrib/alchemy/manager.py

```diff
@@ -256,90 +256,93 @@ def _query_get_first(self, query):
         except NoResultFound:
             raise IndexError()
 
-    def create(self, properties, commit=True):
-        # noinspection properties
-        item = self.model(**properties)
-
-        before_create.send(self.resource, item=item)
-
+    def create(self, properties):
         session = self._get_session()
+        with session.begin_nested():
```

Comment:
there's a double try/except.

The one inside our code, then the one inside the `with` (https://github.com/sqlalchemy/sqlalchemy/blob/6a622c636ca5bc55d96b92652fd43b914a77645c/lib/sqlalchemy/engine/base.py#L1764).

Not to sure we need the one here.